content/en/core/overview/self-hosting-multiple-nodes.md

```diff
@@ -0,0 +1,285 @@
+---
```

Comment:
Please move this to beside the other hosting docs here: https://docs.communityhealthtoolkit.org/apps/guides/hosting/4.x/

Reply:
fixed in 1f7bd06